docs/docs/guides/integrate/login-ui/passkey.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Handle Passkey Registration and Login
+sidebar_label: Passkey
+---
+
+Passkeys are a replacement for passwords that provide faster, easier, and more secure sign-ins to websites and apps across a user’s devices.
+It is based on the [FIDO Standard](https://fidoalliance.org/).
+Unlike passwords, passkeys are always strong and phishing-resistant.
+
+## Flow
+
+![Passkey Registration and Login Flow](/img/guides/login-ui/passkey-flow.png)
+
+There are two different possibilities when you like to enable passkey for your user.
+1. You send the user a notification with a code so the user is able to register passkey on another device
+2. You start the passkey registration directly on the device
+
+## Send Registration Link
+
+When you want to send a link to your user, that enables him to register a new passkey device, you can let ZITADEL send the code or return it in the response so you can send a message by yourself.
+
+Make sure to also include the URL Template to customize the reset link in the email sent to the user, in case of ZITADEL sending the email.a
+
+[Send Registration Link Documentation](/apis/resources/user_service/user-service-create-passkey-registration-link)
```

Comment:
```suggestion
When you want to send a link to your user, that enables him to register a new passkey device, you can choose if ZITADEL sends the code or ZITADEL can return the code in the API response. This way you can send a link through any channel of your choice (email, sms, phone, in-person, postal, ...).

Make sure to include the URL Template that customizes the reset link in the email, in case of ZITADEL sending the email.

[Send Registration Link Documentation](/apis/resources/user_service/user-service-create-passkey-registration-link)
```

Reply:
I do not really understand 
"Make sure to include the URL Template that customizes the reset link in the email, in case of ZITADEL sending the email."

Can you explain what this means?

Reply:
Ok I tweaked the sentence slightly, feel free to add additional context, i.e do we have some definition of what values are available?